src/runtime/pkg/oci/utils.go

```diff
@@ -1202,6 +1205,10 @@ func SandboxConfig(ocispec specs.Spec, runtime RuntimeConfig, bundlePath, cid st
 
 	}
 
+	if sandboxConfig.SandboxResources.WorkloadMemMB < runtime.SandboxWorkloadMemMin {
```

Comment:
I would add to this condition:

if runtime.SandboxWorkloadMemMin != 0

That allows users (and us) to play with smaller sizes.